viability/resources/schemas/dbscripts/postgresql/viability-25.000-25.001.sql

```diff
@@ -0,0 +1 @@
+MiALTER TABLE viability.Results ALTER COLUMN ObjectId TYPE BIGINT;
```

Comment:
Seems like a typo. Should be `ALTER` yea?

Reply:
Dang... typed into wrong window or something